core/node/state_keeper/src/batch_executor/tests/tester.rs

```diff
@@ -356,17 +356,17 @@ impl AccountLoadNextExecutable for Account {
 
     /// Returns a transaction to the loadnext contract with custom amount of write requests.
     /// Increments the account nonce.
-    fn loadnext_custom_writes_call(
+    fn loadnext_custom_new_writes_call(
         &mut self,
         address: Address,
-        writes: u32,
+        new_writes: u32,
```

Comment:
Renamed to reflect that it only supports a custom # of _new_ writes. Maybe it should have an extra argument for overwrites as well?